pkg/config/settings.go

```diff
@@ -22,6 +22,7 @@ const (
 	CSIAttacherImage            = "quay.io/k8scsi/csi-attacher:v1.2.1"
 	CSILivenessProbeImage       = "quay.io/k8scsi/livenessprobe:v1.1.0"
 	CSISnapshotterImage      	= "quay.io/k8scsi/csi-snapshotter:v2.1.0"
+	CSIResizerImage             = "k8s.gcr.io/sig-storage/csi-resizer:v1.0.0"
```

Comment:
Is the csi-resizer (version 1.0.0) exist in quay.io?
If it exist - I think it is better to redirect to quay.io and when we do one commit to move to GCR - it will be contained

Reply:
the gcr PR was [already merged](https://github.com/IBM/ibm-block-csi-operator/pull/107)

Reply:
OK. got it. 10x